gpytorch/variational/lmc_variational_strategy.py

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env python3
+
+import torch
+
+from ..distributions import MultitaskMultivariateNormal
+from ..lazy import KroneckerProductLazyTensor, MatmulLazyTensor
+from ..module import Module
+from ._variational_strategy import _VariationalStrategy
+
+
+class LMCVariationalStrategy(_VariationalStrategy):
+    r"""
+    LMCVariationalStrategy is an implementation of the "Linear Model of Coregionalization"
+    for multitask GPs. This model assumes that there are :math:`Q` latent functions
+    :math:`\mathbf g(\cdot) = [g^{(1)}(\cdot), \ldots, g^{(q)}(\cdot)]`,
+    each of which is modelled by a GP.
+    The output functions (tasks) are linear combination of the latent functions:
+
+    .. math::
+
+        f_{\text{task } i}( \mathbf x) = \sum_{q=1}^Q \mathbf a^{(q)} g^{(q)} ( \mathbf x )
```

Comment:
is this the equation you mean?

Reply:
That's this equation from Alvarez (2012):

<img width="175" alt="Screen Shot 2020-06-17 at 12 59 53 PM" src="https://user-images.githubusercontent.com/824157/84938424-3ac35f00-b0ab-11ea-851a-04cf4c1fef85.png">

It leads to Kronecker structure when the latent functions share the same covariance. I think this is right?

Reply:
that equation is fine; maybe i'm misreading your notation but your `a` only has 1 index not 2?

Reply:
ah yeah i see; there should be an `i` index as well.